src/LuaTransformer.ts

```diff
@@ -5515,6 +5518,9 @@ export class LuaTransformer {
                     this.importLuaLibFeature(LuaLibFeature.WeakSet);
                     return;
             }
+            if (tsHelper.isBuiltinErrorTypeName(name)) {
```

Comment:
Whiteline above this please :D

Reply:
We probably should enforce it after #676 with https://eslint.org/docs/rules/padding-line-between-statements